provider/datagen/Cargo.toml

```diff
@@ -49,9 +49,11 @@ icu_timezone = { workspace = true, features = ["datagen"] }
 icu_compactdecimal = { workspace = true, features = ["datagen"], optional = true }
 icu_displaynames = { workspace = true, features = ["datagen"], optional = true }
 icu_relativetime = { workspace = true, features = ["datagen"], optional = true }
-icu_singlenumberformatter = { workspace = true, features = ["datagen"], optional = true}
+# excluded from 1.4 release
+# icu_singlenumberformatter = { workspace = true, features = ["datagen"], optional = true}
```

Comment:
thought: you can make these dev-dependencies to have testdata continue to include them. then we could keep it in this state until these crates become experimentally publishable.

Reply:
We will want to quickly re-enable them because we want to generate bakeddata for them, not just testdata

Reply:
Do we? We don't really need baked data until we are actually able to use it, at which point the crate can be considered experimental. Currently we *only* have datagen code, for which testdata is enough.

Reply:
I'm not a big fan of the disable/enable dance because we want to keep main in a publishable state.

Reply:
okay I'm sold. I like this because it also gives us a path for how to handle similar situations in the future.